---
title: Research
nav:
  order: 1
  tooltip: Our research projects
---

# {% include icon.html icon="fa-solid fa-microscope" %}Our research

Our group combines wet lab and bioinformatics approaches to explore novel ways to use bacteriophages and their enzymes in clinical settings and for biotechnological applications. In addition, we develop novel computational tools and methods to characterise bacteriophages and other viruses. 

## Bacteriophage Formulations
{% capture content %}
  {% include figure-no-shadow.html image="images/Formulations.png" width = "150px"  %}
{% endcapture %}

{%
  include float.html
  content=content
  flip=false
%}

One of our main research directions is the development of novel formulations for bacteriophage delivery.

Many pathogenic bacteria such as <i>Salmonella, Listeria</i> and <i>Mycobacterium tuberculosis</i> evade the immune system by hiding inside the host cells. In order to reach and eradicate the intracellular bacteria, bacteriophages need to be delivered inside the infected cells. One of the ways to achieve this is to encapsulate bacteriophages in liposomes. This approach not only allows to improve intracellular phage delivery, but also protects bacteriophages from the host immune system and increases their stability in the environment, prlonging the shelf life of liposomal phage formulations.

A promising approach to improve phage delivery is to formulate bacteriophages into dissolvable microneedles. Microneedles are microscopic needles produced from biocompatible polymers that can be used to deliver drugs and vaccines through the skin. A microneedle patch can be self-administered and presents a painless and non-invasive method of drug delivery. As <a href="https://www.biofilms.ac.uk/manage-and-influence-skin-microbiome/">the potential of bacteriophages for regulation of skin microbiome</a> has been recognised by the cosmetic industry, we are currently developing bacteriophage-loaded microneedles for the treatment of acne. An alternative method of transdermal delivery we have also started recently exploring is based on the use of deep eutectic solvents (DES). It has been previously shown that DES can be used to deliver large molecules such as proteins through the epidermal barrier, while retaining their biological activity. We are investigating whether DES can be used to improve the delivery of bacteriophages through the skin to treat skin infections. 

In addition to the above, we are interested in functionalisation of catheter surfaces with bacteriophages and their enzymes to prevent catheter-associated infections, especially urinary tract infections. We mainly focus on immobilisation of bacteriophages in hydrogels and covalent attachment of bacteriophages to plastics activated with cold atmospheric plasma.

Finally, we assess the potential of using phages for elimitaion of biofilms in combinations with other antimicrobial agents, such as antibiotics and cold atmospheric plasma activated water. 

{% include float.html clear=true %}


## Biotechnology and Bioinformatics
{% capture content %}
  {% include figure-no-shadow.html image="images/Bioinformatics.png" width = "150px"  %}
{% endcapture %}

{%
  include float.html
  content=content
  flip=true
%}

Bacteriophages encode a wide range of enzymes that can be used in biotechnology and medicine. In our laboratory we are focussing on two classes of phage enzymes: endolysins and poysaccharide depolymerases. We have recently obtained several soluble and active endolysins from <i>Cutibacterium acnes</i> bacteriophages and are currently investigating their potential for the treatment of acne in combination with other therapeutic agents. Another group of lytic enzymes we are working on are endolysins of <i>Proteus mirabilis</i> -- to the best of our knowledge, we are the first to characterise these enzymes and begin exploring their potential for the treatment of urinary tract infections. 

In 2021, we isolated a <i>Proteus</i> phage forming halos on bacterial lawns, indicating depolymerase activity. We were able to express and purify the depolymerase and demonstrate its ability to degrade the <i>Proteus</i> biofilms and reduce mortality of <i>Proteus</i>-infected <i>Galleria mellonella</i> larvae. 

Despite continuing advances in sequencing technologies and viral genomics, the majority of bacteriophage genes remain uncharacterised. One of our latest projects involved the development of a novel machine learning tool, <a href="https://doi.org/10.1186/s12859-023-05341-w">DePolymerase Predictor (DePP)</a>, for rapid and accurate identification of bacteriophage depolymerases in phage genomic sequences. In contrast to most protein function prediction methods, only experimentally verified sequences were used in DePP's training set. Despite that, we demonstrated that DePP performs comparably to other state-of-the-art tools, and in some cases outperforms them, thus confirming the validity of our approach. We plan to continue the development of DePP and intend to both expand our approach to other phage enzymes and improve the accuracy of the tool by incorporating structural information into the model in the future.



{% include float.html clear=true %}


## Other Research Projects
{% capture content %}
  {% include figure-no-shadow.html image="images/Other_Projects.png" width = "150px"  %}
{% endcapture %}

{%
  include float.html
  content=content
  flip=false
%}

Over the past few years we have been involved in several other research projects focussing on genomics of human and animal pathogens. 

In collaboration with Prof Chris Creevey from Queen's University School of Biological Sciences we have been working on the Phage-SUAS project that  aims to develop innovative and effective phage therapy solutions to specifically target an important pathogen <i>Streptococcus suis</i> in pigs. This involves the analysis of the pangenome of <i>S. suis</i>, exploration of the diversity of <i>S. suis</i> prophages, and prediction of phage-hosts associations.

Since 2021, we have been involved in the SARS-CoV-2 sequencing and genomic analysis in Northern Ireland under the COG-UK Consortium. We are working in collaboration with Prof David Simpson, COG-UK Principal Investigator for Queenâ€™s University Belfast. Our innovative phylogenetic and geospatiotemporal analysis of thousands of SARS-CoV-2 genomic sequences from Northern Ireland allowed to identify the events introducing the virus to Northern Ireland and track the spread of the virus across the island of Ireland. We continue working on this exciting and publlicly important project, and are currently investigating the specifics of SARS-CoV-2 evolution in Northern Ireland.


{% include float.html clear=true %}